include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -61,6 +61,13 @@ static const char* const kOrtSessionOptionsUseDeviceAllocatorForInitializers = "
 static const char* const kOrtSessionOptionsConfigAllowInterOpSpinning = "session.inter_op.allow_spinning";
 static const char* const kOrtSessionOptionsConfigAllowIntraOpSpinning = "session.intra_op.allow_spinning";
 
+// Key for disable copy model bytes for ORT format
+// By default we will copy the model bytes for ORT format at the time of session creation to ensure the model bytes
+// buffer is valid.
+// Setting this option to "1" will disable copy the model bytes. The caller has to guarantee that the model bytes
+// are valid until the ORT session using the model bytes is destroyed.
+static const char* const kOrtSessionOptionsConfigDisableCopyORTModelBytes = "session.disable_copy_ort_model_bytes";
+
```

Comment:
nit: would 'use_ort_model_bytes_directly' be slightly more generic and cover both the copy and potential usage of initializers from the bytes? not a big difference though.

Reply:
Can you also update [https://www.onnxruntime.ai/docs/how-to/mobile/model-execution.html](https://www.onnxruntime.ai/docs/how-to/mobile/model-execution.html) to call out this option?

Reply:
> Can you also update https://www.onnxruntime.ai/docs/how-to/mobile/model-execution.html to call out this option?

Will do in a separated PR